test/test_vtx.py

```diff
@@ -130,3 +131,4 @@ def test_filename_raises_error_when_wrong_type():
     """Test that the filename attribute raises an error if the extension is not .bp"""
     with pytest.raises(TypeError):
         F.VTXSpeciesExport(1, field=[F.Species("H")])
+        F.VTXTemperatureExport(1)
```

Comment:
Since the line before triggers an error, this will not be tested

Please write another test